### DISCUSSION TITLE
"Create New" option in association drop-downs

### DISCUSSION THREAD

**Discussion by nhorton** [avo-hq/avo#2012] [Feature requests]
It would be super-helpful if I could have an option on a `belongs_to` association (but others too really) that would have an option to create a new object of the type of the association instead of choosing an existing one.

**Comment by adrianthedev** [comment]
Is [this](https://github.com/avo-hq/avo/pull/1876) what you mean?

We merged two releases ago.

**Comment by Paul-Bob** [comment]
<img width="960" height="255" alt="image" src="https://github.com/user-attachments/assets/86a764e9-4c1d-4c8f-bbe6-137e94efb4c7" />
